### PR TITLE
fix: dockerfile CMD and missed dependency

### LINKED ISSUE #782: Invalid dockerfiles for the latest master branch.
I am trying to build grobid docker image with the latest master branch.

The problem of dockerfile is the new config is located in `grobid-home/config/grobid.yaml`, not `grobid-service/config/config.yaml`.

```dockerfile
CMD ["./grobid-service/bin/grobid-service", "server", "grobid-service/config/config.yaml"]
```

```
02:17:13.968 [main] INFO org.grobid.service.main.GrobidServiceApplication - Configuration file: /opt/grobid/grobid-service/config/config.yaml
java.io.FileNotFoundException: File grobid-service/config/config.yaml not found
        at io.dropwizard.configuration.FileConfigurationSourceProvider.open(FileConfigurationSourceProvider.java:18)
        at io.dropwizard.configuration.BaseConfigurationFactory.build(BaseConfigurationFactory.java:80)
        at io.dropwizard.cli.ConfiguredCommand.parseConfiguration(ConfiguredCommand.java:126)
        at io.dropwizard.cli.ConfiguredCommand.run(ConfiguredCommand.java:74)
        at io.dropwizard.cli.Cli.run(Cli.java:78)
        at io.dropwizard.Application.run(Application.java:93)
        at org.grobid.service.main.GrobidServiceApplication.main(GrobidServiceApplication.java:126)
```

PS: I think we shouldn't use hardcode arguments.

### PR DESCRIPTION
**Pull request by elonzh**
close #782 

I tested the CMD without arguments it will use default arguments.

```
06:53:10.835 [main] INFO org.grobid.service.main.GrobidServiceApplication - Found conf path: /opt/grobid/grobid-home/config/grobid.yaml
06:53:10.838 [main] WARN org.grobid.service.main.GrobidServiceApplication - Running with default arguments: "server" "/opt/grobid/grobid-home/config/grobid.yaml"
06:53:10.838 [main] INFO org.grobid.service.main.GrobidServiceApplication - Configuration file: /opt/grobid/grobid-home/config/grobid.yaml
```

### CODE REVIEW

**Comment by kermitt2** [comment]
Thanks! 
To be updated too: `doc/Grobid-service.md`
To be updated or better removed: `grobid-service/README.md`

File: Dockerfile.crf [hunk 1]
```diff
@@ -74,7 +74,7 @@ ENV TINI_VERSION v0.18.0
 ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
 RUN chmod +x /tini
 ENTRYPOINT ["/tini", "-s", "--"]
-CMD ["./grobid-service/bin/grobid-service", "server", "grobid-service/config/config.yaml"]
+CMD ["./grobid-service/bin/grobid-service"]
```
Comment (kermitt2) [inline Dockerfile.crf:77]: Actually I am wondering, why not 
```
CMD ["./gradlew grobid-service:run"]
```
and forget about the standalone application?

**Comment by elonzh** [comment]
> Thanks!
> To be updated too: `doc/Grobid-service.md`
> To be updated or better removed: `grobid-service/README.md`

Ok I will update it.

File: Dockerfile.crf (hunk 1 above)
Comment (kermitt2) [reply Dockerfile.crf:77]: Bad idea sorry !

**Review by kermitt2 — APPROVED** [review]

**Comment by elonzh** [comment]
@kermitt2 docs are updated.

**Comment by kermitt2** [comment]
Thank you so much !

**Pull request merged**